### DISCUSSION TITLE
Can you share the images?

### DISCUSSION THREAD

**Discussion by haclongtd1994** [alexanderlavrushko/BLE-HUD-navigation-ESP32#23] [Q&A] [answered]
Hi @alexanderlavrushko,
Could you please share the images which generated 4-bit data arrays.
Eg: IMG_directionWaypoint, IMG_directionEasyLeft, and so on.
Thank you so much

**Comment by alexanderlavrushko** [comment, marked as answer]
Hi! Unfortunately, I cannot. The closest point I have, is this old project sygic-maps-kit-ios, where the images are available as PDF, they appear to be white arrows on transparent background, and the only way I was able to see them is to open folder Directions.xcassets in Xcode, it's located here: https://github.com/Sygic/sygic-maps-kit-ios/tree/master/SygicMapsKit/Resources

<img width="605" alt="Screenshot 2022-10-18 at 18 47 43" src="https://user-images.githubusercontent.com/28007228/196495538-ed0f2ab7-7371-4e00-9afd-fc299f0301dc.png">

Also there are some vector images written in XML for Android, here: https://github.com/Sygic/sygic-maps-kit-android/tree/master/uikit/views/src/main/res/drawable
